src/platform/forms-system/src/js/review/ReviewChapters.jsx

```diff
@@ -32,8 +33,18 @@ class ReviewChapters extends React.Component {
   handleToggleChapter({ name, open, pageKeys }) {
     if (open) {
       this.props.closeReviewChapter(name, pageKeys);
+      if (this.props.form.formId === '22-1995') {
+        recordEvent({
+          event: 'nav-accordion-collapse',
+        });
+      }
```

Comment:
I'm new here and don't have a ton of context yet. Please pardon me if the following is not helpful. :)

Not a blocker, but a question & possible suggestion: Is this a pattern we expect to repeat elsewhere? If so, would it make future work faster, cleaner, and/or easier if we encapsulated this logic in a method that could be invoked like:

`this.props.recordFormEvent(formId, 'nav-accordion-collapse);`